src/sidebar/views/overview.ts

```diff
@@ -16,91 +16,87 @@ export class OverviewItem {
+        ],
+        'normal': [
+            new OverviewItem(() => `Total files analyzed: ${ExtensionApi.metadata.sourceFiles.size}`),
+            new OverviewItem(() => `Last analysis run: ${
+                new Date(ExtensionApi.metadata.metadata!.timestamps.begin /* seconds */ * 1000).toLocaleString()
+            }`),
+            new OverviewItem(() => {
+                // Time in seconds
+                const beginTime = ExtensionApi.metadata.metadata!.timestamps.begin;
+                const endTime = ExtensionApi.metadata.metadata!.timestamps.end;
+                const interval = endTime - beginTime;
+
+                const hours = Math.floor(interval/3600) % 60;
+                const minutes = Math.floor(interval/60) % 60;
+                const seconds = Math.floor(interval) % 60;
+                const ms = Math.floor(interval * 1000) % 1000;
+
+                if (hours > 0 || minutes > 0) {
+                    // H:MM:SS s / M:SS s
+                    const formattedMinutes = hours > 0 ? (`${hours}:${minutes.toString().padStart(2, '0')}`) : minutes;
+                    return `Analysis duration: ${formattedMinutes}:${seconds.toString().padStart(2, '0')} s`;
+                } else {
+                    // S.fff s
+                    return `Analysis duration: ${(seconds + ms / 1000).toFixed(3)} s`;
+                }
+            }),
+            new OverviewItem(() => `Used analyzers: ${
+                Object.keys(ExtensionApi.metadata.metadata!.analyzers).join(', ')
+            }`),
+        ]
     };
 
-    private regularItemsList = [
-        'files',
-        'lastRun',
-        'buildLength',
-        'analyzers',
-
-        'separator',
-
-        'reloadMetadata',
-        'analyzeCurrentFile',
-        'analyzeProject',
-    ];
-    private notFoundItemsList = [
-        'notfound',
-        'notfound2',
-        'notfound3',
+    private bottomItems: {[id: string]: OverviewItem[]} = {
+        'normal': [
+            new OverviewItem('Reload CodeChecker metadata', {
+                title: 'reloadMetadata',
+                command: 'codechecker.backend.reloadMetadata',
+            }),
+            new OverviewItem('Re-analyze current file', {
+                title: 'reloadMetadata',
+                command: 'codechecker.executor.analyzeCurrentFile',
+            }),
+            new OverviewItem('Re-analyze entire project', {
+                title: 'reloadMetadata',
+                command: 'codechecker.executor.analyzeProject',
+            }),
+        ],
+        'ccNotFound': [
+            new OverviewItem('Reload CodeChecker metadata', {
+                title: 'reloadMetadata',
+                command: 'codechecker.backend.reloadMetadata',
+            }),
+            new OverviewItem('——'),
+            new OverviewItem('compile_commands.json not found.'),
+            new OverviewItem('Show database setup dialog...', {
```

Comment:
I clicked on this button first, created the compilation database and if I click to this button again it will show the same pop-up window:
![image](https://user-images.githubusercontent.com/6695818/141765944-c51c0175-2472-4ca6-829d-e473c613981a.png)
It is a little bit missleading because I already have a compilation database.

Reply:
We have to show this item above the `Reload CodeChecker metadata` item because it is more important and it would be better to rename this item something to similar:
```suggestion
            new OverviewItem('Init compilation database', {
```

Reply:
Fixed the button order, and added a new text when there's already a database selected.

I'd rather not rename the button, as I refer to the dialog as the `database setup dialog` in the settings and the README.